llvm/test/CodeGen/RISCV/cheri/global-cap-import-attributes.ll

```diff
@@ -0,0 +1,323 @@
+; RUN: llc --filetype=asm --mcpu=cheriot --mtriple=riscv32-unknown-unknown -target-abi cheriot -mattr=+xcheri,+cap-mode < %s | FileCheck %s
+
+target datalayout = "e-m:e-p:32:32-i64:64-n32-S128-pf200:64:64:64:32-A200-P200-G200"
+target triple = "riscv32cheriot-unknown-cheriotrtos"
+
+%struct.Uart = type { i8 }
+
+
+; Function Attrs: mustprogress noinline optnone
+define dso_local void @_Z7examplev() addrspace(200) #0 {
+entry:
+; CHECK:        .LBB0_1:                                # %entry
+; CHECK-NEXT:                                           # Label of block must be emitted
+; CHECK-NEXT:	        auipcc	ca0, %cheriot_compartment_hi("__import_mem_uart_----")
+; CHECK-NEXT:	        clc	ca0, %cheriot_compartment_lo_i(.LBB0_1)(ca0)
+; CHECK-NEXT:	        ccall	_Z19doSomethingWithUartP4Uart
+  call void @_Z19doSomethingWithUartP4Uart(ptr addrspace(200) noundef @uart_no_perm)
```

Comment:
Please can you add a test with multiple references to the same imported global?

Reply:
2b868bf1809d79df52e2b920ec41246a80d2f34c adds checks to multiple calls to the same imported global.